jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -162,8 +162,8 @@ export class SortingViewModel extends DOMWidgetModel {
       pluginName: '',
       sortingObject: {},
       recordingObject: {},
-      curation: defaultSortingCuration,
-      selection: defaultSortingSelection
+      curation: {},
```

Comment:
No longer maintain a defaultSortingCuration and a defaultSortingSelection. Just use empty object for default, and all fields are optional.